plugins/modules/luks_device.py

```diff
@@ -217,6 +217,47 @@
             - "Will only be used on container creation."
         type: int
         version_added: '1.5.0'
+    perf_same_cpu_crypt:
+        description:
+            - "This option allows user to perform encryption using the same cpu that IO was submitted on."
+            - "The default is to use an unbound workqueue so that encryption work is automatically balanced between available CPUs."
+            - "This option is only relevant for open action."
```

Comment:
```suggestion
            - "Will only be used when opening containers."
```

Reply:
I used exact phrases from `man cryptsetup` so why change this?

Reply:
Because (a) you cannot simply copy documentation from somewhere else (copyright, license, and potentially other legal fineprint), and (b) that's the way this is formulated in other places in this module.